homework/classes/zadania_4.3_4.5.py

```diff
@@ -0,0 +1,176 @@
+### Zadanie 4.3 | Pociąg
+import pytest
+
+
+class Train:
+    MAX_SPEED_UPDATE = 75
+
+    def __init__(self, speed: int, fuel_amount: float):
+        self.speed = speed
+        self.fuel_amount = fuel_amount
+
+    def desc(self) -> str:
+        return f'Moja predkość to {self.speed}. Mam jeszcze {self.fuel_amount} litrów paliwa.'
+
+    def __str__(self) -> str:
+        return self.desc()
+
+    def acceleration(self, increase_speed: int) -> (int, float):
```

Comment:
Co do nazewnictwa, ja bym zmienił na `accelerate`, bo to wskazuje, że zmienia się stan obiektu.

Reply:
zgoda, poprawiam